pandas/core/arrays/boolean.py

```diff
@@ -59,7 +59,7 @@ class BooleanDtype(BaseMaskedDtype):
     name = "boolean"
 
     @property
-    def type(self) -> Type[np.bool_]:
+    def type(self) -> Type:
```

Comment:
@simonjayhawkins I get a mypy error here saying "pandas/core/arrays/boolean.py:62: error: Signature of "type" incompatible with supertype "BaseMaskedDtype"  [override]"

But the base class version *also* uses `Type` as return type (only as a class attribute (``type: Type``) instead of property). Do you know how to "solve" this warning?

Reply:
This is a mypy issue. https://github.com/python/mypy/issues/4125

so could add an ignore and `https://github.com/python/mypy/issues/4125` as a comment

Reply:
@simonjayhawkins thanks! That indeed looks that issue, so will add an ignore with a comment then.